ui/org.eclipse.pde.ui.templates/templates_3.1/browserView/java/$className$.java

```diff
@@ -54,7 +54,7 @@ public class $className$ extends ViewPart implements ISelectionListener {
 
 	@Override
 	public void createPartControl(Composite parent) {
-		fBrowser = new Browser(parent, SWT.WEBKIT);
```

Comment:
@akurtakov should this style bit load internet explorer or does it need fixing in SWT?

Reply:
The default is being changed from IE to Edge during the current release cycle.

https://www.eclipse.org/lists/cross-project-issues-dev/msg19996.html

Reply:
IMO the template should not choose any browser engine at all.

Reply:
Yes, sorry for not giving clearer answer.